### ISSUE TITLE
Main.JS is CC-BY-NC

### ISSUE THREAD

**Issue by ghost** [rosedu/cdl.rosedu.org#40]
This repository contains several files called `js/main.js` across a few branches, and across several different directories per branch.
I am the original author of `js/main.js`, although due to that file having been copied many times over, in some places `git blame` shows a different author for it.

You can recognize `js/main.js` by the fact that it contains some or all of the following distinctive snippets of code:

`function navigate(visualstate) {`

`$(".visualstate").hide();`

`$(css_selector_to_display).css({"display": "block", "opacity": 0}).animate({"opacity": 1}, 250);`

`$(".cdlmenuitem").on("click", function(e) {`

`$(window).on("popstate", function(event) {`

`Zepto(function($) {`

Whenever you encounter a JS file with some or all of the above listed contents, please be reminded that the intended license for it is CC-BY-NC with clarifications, just like for its Upstream Challenge counterpart.

**Comment by tudalex** [comment]
Sorry but I don't see the issue here, because this repo doesn't have a global license file. Can you rephrase? 
Can you also please add a license reference to the files themselves in a comment at the top of the file?
That would make it easier to administer licensing. 

Also regarding code and CC-BY-NC you should read the FAQ at Creative Commons website and maybe consider another license that was meant for code, not art.
